libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -349,14 +350,42 @@ void SITL_State::_simulator_servos(SITL::Aircraft::sitl_input &input)
         wind_speed =     _sitl->wind_speed_active     = (0.95f*_sitl->wind_speed_active)     + (0.05f*_sitl->wind_speed);
         wind_direction = _sitl->wind_direction_active = (0.95f*_sitl->wind_direction_active) + (0.05f*_sitl->wind_direction);
         wind_dir_z =     _sitl->wind_dir_z_active     = (0.95f*_sitl->wind_dir_z_active)     + (0.05f*_sitl->wind_dir_z);
+        
+        // give 5 seconds to calibrate airspeed sensor at 0 wind speed
+        switch (wind_start_delay) {
```

Comment:
What we usually do for this type of code is just check that the micros is zero and set it in that case, then do the if check; something like:

```
if (wind_start_delay_micros == 0) {
   wind_start_delay_micros = now;
} else if ((now - wind_start_delay_micros) >= 5000000) {
    switch (_sitl->wind_type) {
    ...
    }
}
```

That's much smaller, cleaner and uses one less variable.